scripts/wait-nevermined.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
In this case we should be able to remove `scripts/wait_for_migration_and_extract_keeper_artifacts.sh`, right?